.github/workflows/linux-qe-test.yml

```diff
@@ -10,21 +10,13 @@ jobs:
   linux-qe:
     if: | 
       github.event.workflow_run.conclusion == 'success' &&
-      github.event.workflow_run.event == 'pull_request' &&
       github.event.workflow_run.triggering_actor != 'dependabot[bot]'
-    uses: crc-org/crc/.github/workflows/linux-qe-template.yml@main
+    uses: lilyLuliu/crc/.github/workflows/linux-qe-template.yml@main
```

Comment:
Need to change this before merge

Reply:
The testing farm upgrade again that bastion was removed for connection to the reserved machine. So I made some tests on my repo. The has changed back.